Language/ja/Number.php

```diff
@@ -11,19 +11,24 @@
 
 // Number language settings
 return [
-    'terabyteAbbr' => 'TB',
-    'gigabyteAbbr' => 'GB',
-    'megabyteAbbr' => 'MB',
-    'kilobyteAbbr' => 'KB',
-    'bytes'        => 'Bytes',
+    'terabyteAbbr' => 'TB', // 'TB'
+    'gigabyteAbbr' => 'GB', // 'GB'
+    'megabyteAbbr' => 'MB', // 'MB'
+    'kilobyteAbbr' => 'KB', // 'KB'
+    'bytes'        => 'Bytes', // 'Bytes'
 
     // don't forget the space in front of these!
     // Japanese numeral are ten-thousand-based, cause it can not adapt 'number_to_amount()' as below.
     // 日本語の位取りは万進法なのでnumber_to_amount()の動作に適応するのは困難なためローカライズしません。
-    // あえて使いたい人は、完全ではありませんがコメントの内容に置き換えることも可能です
-    'thousand'    => ' thousand', // thousand → '千'
-    'million'     => ' million', // million → '百万'
-    'billion'     => ' billion', // billion → '0億' 1 billionは 10億 ただし、カンマ位取りがおかしくなります（1,0000億というような表示になります）
-    'trillion'    => ' trillion', // trillion → '兆'
-    'quadrillion' => ' quadrillion', // quadrillion → ',000兆' 京としたいところだが 1 quadrillion は 1000兆なので
+    // あえて使いたい人は、完全ではありませんが以下の内容に置き換えることも可能です。
+    // thousand → '千'
+    // million → '百万'
+    // billion → '0億' 1 billionは 10億 ただし、カンマ位取りがおかしくなります（1,0000億というような表示になります）
+    // trillion → '兆'
+    // quadrillion → ',000兆' 京としたいところだが 1 quadrillion は 1000兆なので
+    'thousand'    => ' thousand', // ' thousand'
+    'million'     => ' million', // ' million'
+    'billion'     => ' billion', // ' billion'
+    'trillion'    => ' trillion', // ' trillion'
+    'quadrillion' => ' quadrillion', // ' quadrillion'
```

Comment:
Curious why the English variant?

Reply:
They are not translated because translating is difficult due to the different numbering system.
Not three digits, but four digits.

1000 → 1千
1 0000 → 1万
1 0000 0000 → 1億
1 0000 0000 0000 → 1兆
1 0000 0000 0000 0000 → 1京

If we need localized `number_to_amount()`, we need to extend the function.